hw/ip/spi_device/README.md

```diff
@@ -94,15 +70,14 @@ Mailbox is an exception as it shares the Read command opcode.
 
 ### SPI Device Modes and Active Submodules
 
-SPI Device HWIP has three modes + TPM mode, which are "Generic" mode (also known as FwMode), "Flash" mode, and "Passthrough" mode.
-Generic mode is exclusive. Flash and Passthrough modes share many parts of the datapath.
-TPM mode only shares the SPI and has separate CSb port, which allows that the host sends TPM commands while other SPI mode is active (except Generic mode).
+SPI Device HWIP has two modes + TPM mode, which are "Flash" mode and "Passthrough" mode.
```

Comment:
Nit. this sentence was already a bit strange, could you make it a bit more sensible since you're here?